src/common/pure/InlineBanner/cosmos.decorator.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useCallback, useContext, ReactNode } from 'react'
+import { ThemeContext } from 'styled-components/macro'
+import { useDarkModeManager } from 'state/user/hooks'
+
+const DarkModeToggle = ({ children }: { children?: ReactNode }) => {
+  const theme = useContext(ThemeContext)
+  const [darkMode, toggleDarkModeAux] = useDarkModeManager()
+  const toggleDarkMode = useCallback(() => {
```

Comment:
for this probably will be nicer to do something more global, like a decorator.

happy to help with this one, let me know!
https://github.com/react-cosmos/react-cosmos/blob/main/docs/README.md#decorators

Reply:
A separate decorator than this one?